src/Tkmm/Resources/locales.json

```
@@ -1650,6 +1650,50 @@
       "zh_CN": "",
       "zh_TW": ""
     },
+    "SetupWizard_MissingKeys_Title": {
+      "ar_SA": "",
+      "de_DE": "",
+      "el_GR": "",
+      "en_US": "Keys are missing",
```

Titles should use title case.

```suggestion
      "en_US": "Missing Keys",
```